docs/api-content/api-docs/3-postman-collection.md

```diff
@@ -35,7 +35,7 @@ Learn more about [authentication methods](/user-management/authentication/api-ke
 
 Use the following URL to import your configuration into Postman:
 
-https://raw.githubusercontent.com/spectrocloud/librarium/master/docs/api-content/api-docs/palette-apis.json <br />
+https://raw.githubusercontent.com/spectrocloud/librarium/version-4-3/docs/api-content/api-docs/v1/api.json
```

Comment:
```suggestion
https://raw.githubusercontent.com/spectrocloud/librarium/version-4-3/docs/api-content/api-docs/palette-apis.json
```

Which one is the correct one? 
Also, in the first sentence of this page it says Spectro Cloud API instead of Palette API (I could not leave a comment idw why)